src/release/breaking-changes/2-5-deprecations.md

```diff
@@ -0,0 +1,346 @@
+
+API documentation:
+* [`FloatingHeaderSnapConfiguration`][]
+* [`SliverPersistentHeaderDelegate`][]
+* [`TickerProvider`][]
+
+Relevant issues:
+* [Issue 25507]({{site.repo.flutter}}/issues/25507)
+
+Relevant PRs:
+* Deprecated in [#56413]({{site.repo.flutter}}/pull/56413)
+* Removed in [#90293]({{site.repo.flutter}}/pull/90293)
+
+[Control SliverPersistentHeader's showOnScreen Behavior]: https://docs.google.com/document/d/1BZhxy176uUnqOCnXdnHM1XetS9mw9WIyUAOE-dgVdUM/edit?usp=sharing
+[`FloatingHeaderSnapConfiguration`]: {{site.api}}/flutter/rendering/FloatingHeaderSnapConfiguration-class.html
+[`SliverPersistentHeaderDelegate`]: {{site.api}}/flutter/widgets/SliverPersistentHeaderDelegate-class.html
+[`TickerProvider`]: {{site.api}}/flutter/scheduler/TickerProvider-class.html
+
+---
+
+### `AndroidViewController` & subclasses' `id`
+
+Supported by Flutter Fix: yes
+
+The `id` of `AndroidViewController`, `TextureAndroidViewController`, and
+`SurfaceAndroidViewController`, was deprecated in v1.20.
+
+For all of these use cases, `viewId` should be used instead.
+
+**Migration guide**
+
+Code before migration:
+<!-- skip -->
+```dart
+final SurfaceAndroidViewController surfaceController = SurfaceAndroidViewController(
+  viewId: 10,
+  viewType: 'FixTester',
+  layoutDirection: TextDirection.ltr,
+);
+int viewId = surfaceController.id;
+final SurfaceAndroidViewController surfaceController = SurfaceAndroidViewController(
+  error: '',
+);
+final TextureAndroidViewController textureController = TextureAndroidViewController(
+  error: '',
+);
+final TextureAndroidViewController textureController = TextureAndroidViewController(
+  viewId: 10,
+  viewType: 'FixTester',
+  layoutDirection: TextDirection.ltr,
+);
+viewId = textureController.id;
+```
+
+Code after migration:
+<!-- skip -->
+```dart
+final SurfaceAndroidViewController surfaceController = SurfaceAndroidViewController(
+  viewId: 10,
+  viewType: 'FixTester',
+  layoutDirection: TextDirection.ltr,
+);
+int viewId = surfaceController.viewId;
+final SurfaceAndroidViewController surfaceController = SurfaceAndroidViewController(
+  error: '',
+);
+final TextureAndroidViewController textureController = TextureAndroidViewController(
+  error: '',
+);
+final TextureAndroidViewController textureController = TextureAndroidViewController(
+  viewId: 10,
+  viewType: 'FixTester',
+  layoutDirection: TextDirection.ltr,
+);
+viewId = textureController.viewId;
+```
+
+**References**
+
+Design document:
+* [Flutter Hybrid Composition][]
+
+API documentation:
+* [`AndroidViewController`][]
+* [`TextureAndroidViewController`][]
+* [`SurfaceAndroidViewController`][]
+
+Relevant issues:
+* [Issue 55218]({{site.repo.flutter}}/issues/55218)
+
+Relevant PRs:
+* Deprecated in [#60320]({{site.repo.flutter}}/issues/60320)
+* Removed in [#90294]({{site.repo.flutter}}/issues/90294)
+
+<!-- TODO - design doc link broken in #60320 -->
+[Flutter Hybrid Composition]: /go/something
```

Comment:
@bparrishMines is there a chance you have this link to the design doc? The link is broken in the original PR.

Reply:
Is this what you were thinking of: https://github.com/flutter/flutter/wiki/Hybrid-Composition

Reply:
Thanks! That is it! :)